Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -84,6 +84,15 @@ CTLineRef CTTypesetterCreateLineWithOffset(CTTypesetterRef ts, CFRange range, do
     RETURN_NULL_IF(!frame);
     THROW_NS_IF_FALSE(E_UNEXPECTED, [frame->_lines count] == 1);
 
+    if ([frame->_lines count] != 1) {
```

Comment:
oops, I thought I did that...

---
In reply to: [91176623](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r91176623) [](ancestors = 91176623)